public/jira/rollup/percent-complete/percent-complete.test.ts

```diff
@@ -0,0 +1,813 @@
+              derivedTiming: { totalDaysOfWork: 1, completedDaysOfWork: 0 },
+            },
+            {
+              parentKey: "i-1",
+              key: "e-3",
+              derivedTiming: { totalDaysOfWork: 4, completedDaysOfWork: 2 },
+            },
+          ],
+          [
+            {
+              parentKey: "e-2",
+              key: "s-4",
+              derivedTiming: { totalDaysOfWork: 8, completedDaysOfWork: 3 },
+            },
+            {
+              parentKey: "e-2",
+              key: "s-5",
+              derivedTiming: { totalDaysOfWork: 16, completedDaysOfWork: 12 },
+            },
+            {
+              parentKey: "e-3",
+              key: "s-6",
+              derivedTiming: { totalDaysOfWork: 32, completedDaysOfWork: 19 },
+            },
+            {
+              parentKey: "e-3",
+              key: "s-7",
+              derivedTiming: { totalDaysOfWork: 64, completedDaysOfWork: 31 },
+            },
+          ],
+        ] as IssueOrRelease[][]
+      ).reverse();
+
+      const expected = [
+        {
+          rollupData: [
+            {
+              completedWorkingDays: 3,
+              remainingWorkingDays: 5,
+              totalWorkingDays: 8,
+              userSpecifiedValues: true,
+            },
+            {
+              completedWorkingDays: 12,
+              remainingWorkingDays: 4,
+              totalWorkingDays: 16,
+              userSpecifiedValues: true,
+            },
+            {
+              completedWorkingDays: 19,
+              remainingWorkingDays: 13,
+              totalWorkingDays: 32,
+              userSpecifiedValues: true,
+            },
+            {
+              completedWorkingDays: 31,
+              remainingWorkingDays: 33,
+              totalWorkingDays: 64,
+              userSpecifiedValues: true,
+            },
+          ],
+        },
+        {
+          rollupData: [
+            {
+              completedWorkingDays: 15,
+              remainingWorkingDays: 9,
+              totalWorkingDays: 24,
+              userSpecifiedValues: true,
+            },
+            {
+              completedWorkingDays: 50,
+              remainingWorkingDays: 46,
+              totalWorkingDays: 96,
+              userSpecifiedValues: true,
+            },
+          ],
+        },
+        {
+          rollupData: [
+            {
+              completedWorkingDays: 65,
+              remainingWorkingDays: 55,
+              totalWorkingDays: 120,
+              userSpecifiedValues: true,
+            },
+          ],
+        },
+      ];
+
+      const actual = rollupPercentComplete(issuesAndReleases);
+
+      expect(actual).toHaveLength(expected.length);
+      for (const i of Array(actual.length).keys()) {
+        expect(actual[i].rollupData).toStrictEqual(expected[i].rollupData);
+      }
+    });
+
+    it("should use average estimates if no estimate provided for stories", () => {
+      const issuesAndReleases = (
```

Comment:
I know you're following what I created, but I think we might want to reverse the initial data so its order matches the output.

It's a bit strange that the setup data is "top down" (initiatives to stories), but the data we get out and check against is "bottom up" (stories to initiatives). 

Stranger still, we call `.reverse()` on the setup data.

I bet chat gpt can reverse it for you easily ...